docs/docs/develop/glob-patterns.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Glob Patterns
+description: Support for importing data sources using glob patterns
+sidebar_label: Glob patterns
+sidebar_position: 10
```

Comment:
Should be higher since this is an advanced tutorial and should probably come after the three basic tutorials in this section

You can use `npm run dev -w docs` on local to preview the docs